app/commands/github/issue/search.rb

```diff
@@ -0,0 +1,75 @@
+module Github
+  class Issue::Search
+    include Mandate
+
+    # Use class method rather than constant for
+    # easier stubbing during testing
+    def self.requests_per_page
+      20
+    end
+
+    def initialize(action: nil, knowledge: nil, module_: nil, size: nil, type: nil, repo: nil, order: nil, page: 1)
```

Comment:
The `module` keyword is reserved. What would your preferred name be?

Reply:
`area`?